src/site/layout/header.rs

```diff
@@ -23,140 +19,12 @@ async fn fetch_logo(
     path_prefix: &Option<String>,
     dist_dir: &str,
     origin_path: String,
-    name: &String,
-) -> Result<Box<img<String>>> {
+    _name: &str,
+) -> Result<String> {
```

Comment:
Does some later code properly apply this now..? (maybe rename the function to get_logo_url if so..?)